templates/core/core-dpl.yaml

```diff
@@ -190,9 +195,6 @@ spec:
       - name: auth-ca-cert
         secret:
           secretName: {{ .Values.uaaSecretName }}
-          items:
-            - key: ca.crt
-              path: auth-ca.crt
```

Comment:
why the line193 - line195 is removed?

Reply:
Fixed in [`ce8e201` (#1434)](https://github.com/goharbor/harbor-helm/pull/1434/commits/ce8e20195e5f87386f38c90521f4dd7d4bc3428f)